polaris.shopify.com/src/components/PolarisExampleWrapper/PolarisExampleWrapper.tsx

```diff
@@ -4,7 +4,7 @@ import {ComponentType} from 'react';
 import styles from './PolarisExampleWrapper.module.scss';
 
 const stylesheetHref =
-  'https://unpkg.com/@shopify/polaris@latest/build/esm/styles.css';
+  'https://unpkg.com/@shopify/polaris@0.0.0-snapshot-release-20220922222021/build/esm/styles.css';
```

Comment:
Yup, it's not something we want to commit, I included `TEMP COMMIT -` in the commit message for these. I left them so that reviewers could pull down and run locally but was going to drop them before I merged 👍